fictadvisor-api/src/v2/api/services/GroupService.ts

```diff
@@ -291,11 +318,7 @@ export class GroupService {
       },
     });
 
-    if (!captain) {
-      throw new AbsenceOfCaptainException();
-    }
-
-    return captain.user;
+    return captain?.user;
```

Comment:
This change causes TypeError on groups/:groupId/captain Get endpoint. If group has no captain, undefined is passed to userMapper.getUser method. So, for example, create a separate method in GroupService with this validation for handling that route.